### ISSUE TITLE
Check user rights

### ISSUE THREAD

**Issue by sbancal** [EPFL-ENAC/epfl-enac-deploy-action#3]
https://github.com/EPFL-ENAC/epfl-enac-deploy-action/blob/f1835bb0b42f4e03ee7dca7e62eb823cc53cc0c1/action.yml#L14

We need to test and validate if a non-EPFL-ENAC user that push the commit. Will he be allowed to login !?

**Comment by blueur** [comment]
non-EPFL-ENAC user should not be able to pull Docker image. To be tested.

**Comment by sbancal** [comment]
This question is outdated now with new CD.
